### PR TITLE
Luk10114/integrated mesh layer

### PR DESCRIPTION
**Pull request by LDuncAndroid**
Issue here: https://devtopia.esri.com/runtime/common-samples/issues/1150

### CODE REVIEW

File: java/integrated-mesh-layer/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+scene service.
+3. Add the layer to the scene's operational layers collection.
+4. Create a `Camera` and set the `SceneView`'s viewpoint.
+
+## Relevant API
+
+- IntegratedMeshLayer
+
+## Tags
+
```
Comment (TADraeseke) [inline java/integrated-mesh-layer/README.md]: This line needs to have just "Layers" on it. It's used to define the category of the sample.

It's an inelegant, but convenient way to easily change categories at the moment.

File: java/integrated-mesh-layer/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">Integrated Mesh Layer</string>
```
Comment (TADraeseke) [inline java/integrated-mesh-layer/src/main/res/values/strings.xml]: We're moving towards sentence case on everything in samples Doc at the moment

File: java/settings.gradle [hunk 3]
```diff
@@ -1,4 +1,4 @@
-include ':add-graphics-renderer',
+include ':add-graphics-renderer', ':integrated-mesh-layer',
```
Comment (TADraeseke) [inline java/settings.gradle]: Android Studio isn't clever enough to alphabetise this stuff. Could you slot it in below please?

File: kotlin/integrated-mesh-layer/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Add an integrated mesh layer
+
+View an integrated mesh layer from a scene service.
+
+![Integrated Mesh Layer](integrated-mesh-layer.png)
+
+## Use Case
```
Comment (TADraeseke) [inline kotlin/integrated-mesh-layer/README.md]: Changes as per other README

File: kotlin/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.kt [hunk 5]
```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *  * Copyright 2019 Esri.
```
Comment (TADraeseke) [inline kotlin/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.kt]: copyright formatting as per Java sample

File: kotlin/integrated-mesh-layer/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">Integrated Mesh Layer</string>
```
Comment (TADraeseke) [inline kotlin/integrated-mesh-layer/src/main/res/values/strings.xml]: sentence case

File: java/integrated-mesh-layer/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Add an integrated mesh layer
+
+View an integrated mesh layer from a scene service.
+
+![Integrated Mesh Layer](integrated-mesh-layer.png)
+
+## Use Case
+
+An integrated mesh can represent artificial and natural 3D features such as
```
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply java/integrated-mesh-layer/README.md]: Should this be updated in the designs before I make the change as should it not be relfected in the other platofrms?

File: java/integrated-mesh-layer/README.md (hunk 1 above)
Comment (LDuncAndroid) [reply java/integrated-mesh-layer/README.md]: I've just copied what the Design README defined, but shouldn't each plaform match as close as possible?

https://devtopia.esri.com/runtime/common-samples/blob/master/designs/AddAnIntegratedMeshLayer/README.md

File: java/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.java [hunk 8]
```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *  * Copyright 2019 Esri.
```
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply java/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.java]: I was using the `Copyright` tool in Android Studio. Unfortunately using this doesn't allow the Copyright statement on the opening line. I was also using the sample found here: https://devtopia.esri.com/runtime/java-common-api/wiki/Java-and-Android-code-guidelines which doesn't seem to match the one in the Samples Wiki.

[the remark this answers is not in the document]
Comment (TADraeseke) [reply java/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.java]: Hmm yeah. Just stick to the one in here: https://devtopia.esri.com/runtime/common-samples/wiki/Android-Samples-Repo%3A-A-Contribution-Guide#copyright

Not overly important as long as its Apache License and no need to have 2 columns of *s

File: java/integrated-mesh-layer/README.md (hunk 1 above)
Comment (TADraeseke) [reply java/integrated-mesh-layer/README.md]: Tags are still a comparatively new thing in READMEs which haven't been fully sorted on all platforms. Just put the tags on the following lines. The Android sample viewer build script reads the first tag and uses it for the category

So:

#### Tags
Layers
3D
integrated mesh

File: java/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (LDuncAndroid) [reply java/integrated-mesh-layer/src/main/java/com/esri/arcgisruntime/sample/integratedmeshlayer/MainActivity.java]: If it's OK with you, I'll stick with the AS tool, which will move the Copyright statement onto the 2nd line. Just don't want to rely upon myself copy and pasting the statement everytime.

File: java/integrated-mesh-layer/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (TADraeseke) [reply java/integrated-mesh-layer/README.md]: Design PR is made. In the mean time lets slot in the text below and we can always go back and update it based on PR feedback:

> An integrated mesh is used to generate a mesh over a point cloud data set, which may be derived from a detailed LiDAR survey. LiDAR data is often of a high enough resolution to capture 3D features as small as walls and trees, but also the irregular surfaces of geological features like cliffs. An integrated mesh made from a point cloud of LiDAR data will include realistic textures and elevation information.

**Comment by TADraeseke** [comment]
@LDuncAndroid - Lets just slot in the proposed new use case text above and we can circle back and update it based on common sample review

**Comment by TADraeseke** [comment]
Good to merge after that, then we'll contact @saratchandrakarumuri or @hardikc07 to verify for us.

**Comment by LDuncAndroid** [comment]
@TADraeseke done.

**Pull request merged**